frontend/src/components/GDXForm/ChipNav/index.tsx

```diff
@@ -15,36 +21,46 @@ export const ChipNav = ({ navLinks }: IChipNav) => {
     textDecoration: "none",
   }));
 
+  /**
+   * Renders a single ChipNav link.
+   *
+   * @param   {IChipNav}    link The ChipNav link to render.
+   * @returns {JSX.Element}
+   */
+  const renderNavLink = (link: IChipNav): JSX.Element => {
+    return (
+      <StyledListItem key={link.key}>
+        <StyledNavLink to={link.url} end>
+          {({ isActive }) => (
+            <Chip
+              sx={{
+                backgroundColor: isActive ? "#D3D3D3" : bcgovTheme.palette.primary.main,
```

Comment:
shouldn't all colors be part of the bcgovTheme?

Reply:
Seems like they should but this is how it was originally so I've just left it the way it was.